### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by Kludex**
@tomchristie Are the steps too complicated? :thinking: 

Inspired by https://github.com/tiangolo/fastapi/security/policy.

### CODE REVIEW

**Comment by gnat** [comment]
```
Security is very important for Starlette. 🔒

Learn more about it below. 👇

## Versions

The latest version of Starlette is supported.

You are encouraged to write tests for your application, and update your Starlette version frequently after ensuring that your tests are passing.
This way you will benefit from the latest features, bug fixes, and **security fixes**.
```
This is all redundant language and could be removed.

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+Learn more about it below. 👇
+
+## Versions
+
+The latest version of Starlette is supported.
+
+You are encouraged to write tests for your application, and update your Starlette version frequently after ensuring that your tests are passing.
+This way you will benefit from the latest features, bug fixes, and **security fixes**.
+
```
Comment (Kludex) [inline SECURITY.md]:
```suggestion
```

**Comment by Kludex** [comment]
ping @tomchristie

File: SECURITY.md (hunk 1 above)
Comment (lovelydinosaur) [reply SECURITY.md]: Agree with dropping this.
Really all we want to do here is outline a process for reporting potential security issues.

Comment (Kludex) [reply SECURITY.md]: committed. 🙏

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+If you think you found a vulnerability, and even if you are not sure about it, please send a message on
+[this discussion](https://github.com/encode/starlette/discussions/1861) sharing how we can contact you (e.g. your email).
+
+We'll contact you as soon as possible, with a proof that we are who we claim we are (you can also validate it 
+[here](https://github.com/orgs/encode/teams/operations)). Then you can reply with the vulnerability.
+
+Please try to be as explicit as possible, describing all the steps and example code to reproduce the security issue.
+
+We will review it thoroughly and get back to you.
```
Comment (lovelydinosaur) [inline SECURITY.md:13]: What do we think *is* the simplest way to have folks chat to us in a private(wish) way?

* A security email address?
* Contact one or more of us privately in our chat space? https://gitter.im/encode/community
* Comment on an issue with their contact details?
* Comment on an issue, requesting an invite, and then we temporarily add them to a security team and discuss on GitHub.

I'm thinking that the chat space might be an okay low-effort approach.
What do other teams in similar spaces to us do?

Comment (Kludex) [reply SECURITY.md:13]: Flask: https://github.com/pallets/flask/security/policy - security email address
Django: https://docs.djangoproject.com/en/dev/internals/security/ - security email address

👀

Comment (gnat) [reply SECURITY.md:13]: We've already had some security issues in github discussions.. allows us to vet if the issue is serious or not, but for a serious exploit maybe PM a dev?

**Comment by Kludex** [comment]
It's possible to create a SECURITY.md for all repositories in the organization level. Let's do that.

**Pull request closed without merging**